index.html

```diff
@@ -0,0 +1,841 @@
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/28.jpg" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/29.jpg" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/4.png" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/4.png" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/4.png" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+        <article class="dflex article-flex-column latest-news-aritcle-row">
+          <img src="./images/4.png" alt="article">
+          <div>
+            <h2>The Ultimate Design Assets Bundle: Lifetime Subscription
+            </h2>
+            <small class="font-small-2x"><span class="story-dealPrice-old">$12189</span> $49.99</small>
+          </div>
+        </article>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <section class="footer-section-1">
+      <div class="dflex footer-section-1-div1 footer-links">
+        <div class="social-icons"> <i class="fab fa-facebook"></i></div>
+        <div class="social-icons"> <i class="fab fa-twitter"></i> </div>
+        <div class="social-icons"> <i class="fab fa-youtube"></i> </div>
+        <div class="social-icons"> <i class="fab fa-instagram-square"></i> </div>
+        <div class="social-icons"> <i class="fab fa-pinterest-p"></i> </div>
+        <div class="social-icons"> <i class="fas fa-envelope"></i> </div>
```

Comment:
The social icons should be links. Wrap them with `a` tags.

Reply:
- [ ] Please, these icons should be links, make sure to wrap them with `a` tags.